quantities/quantities/Area.cs

```diff
@@ -11,42 +11,42 @@ namespace Quantities.Quantities;
     , IMultiplyOperators<Area, Length, Volume>
     , IDivisionOperators<Area, Length, Length>
 {
-    private static readonly Measures.IInject<Quant> square = new RaiseTo<Square>();
-    private static readonly Measures.IInject<Quant> linear = new ToLinear();
-    private readonly Quant quant;
-    internal Quant Quant => this.quant;
-    Quant IQuantity<Area>.Value => this.quant;
-    public Quadratic<To, Area, IArea, ILength> To => new(new To(in this.quant));
-    private Area(in Quant quant) => this.quant = quant;
+    private static readonly Measures.IInject<Quantity> square = new RaiseTo<Square>();
+    private static readonly Measures.IInject<Quantity> linear = new ToLinear();
+    private readonly Quantity area;
```

Comment:
Readability just went a notch higher :grin:

Reply:
Thanks for the feedback :-)